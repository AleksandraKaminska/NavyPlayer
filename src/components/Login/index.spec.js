import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import { Login } from './index'
import renderer from 'react-test-renderer'

Enzyme.configure({ adapter: new Adapter() })

describe('Login', () => {
  it('should render self', () => {
    const rendered = renderer.create(
      <Login track={{}} flow={[]} chosenPlaylist={0} />
    )
    expect(rendered.toJSON()).toMatchSnapshot()
  })
})

describe('>>>Login --- Shallow Render REACT COMPONENTS', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(<Login track={{}} flow={[]} chosenPlaylist={0} />)
  })

  it('+++ contains button', () => {
    expect(wrapper.find('button.login').exists()).toBe(true)
  })
})
