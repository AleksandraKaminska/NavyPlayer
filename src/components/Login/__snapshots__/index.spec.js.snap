// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login should render self 1`] = `
<button
  className="login"
  onClick={[Function]}
>
  Log In
</button>
`;
